eda_analysis/eda_analysis.py

```diff
@@ -54,9 +54,40 @@ def describe_cat_var(dataframe,cat_vars):
     'type':['Car','Bus','Car']
     'height':[10,20,30]
     })
-    >>>cat_vars = ['type']
+    >>> cat_vars = ['type']
     >>> describe_cat_variable(X,cat_vars)
        
     """
     # Code 
     
+def describe_num_var(dataframe, num_vars):
+    """ 
+    This function takes data frame and numeric variable names and provides 
+    statistical summary of the numeric variables for a dataframe.
+    Also, the function plots the histogram of each numeric variable.
+
+    Parameters:
+    -----------
+    dataframe: pandas.DataFrame
+        The dataframe to be inspected.
+    num_vars: list
+        A list of character strings of the names of the numeric variables.
+    
+    Returns:
+    --------
+    dataframe: pandas.DataFrame
+        statistical summary of the numeric variables
+    altair
+        a grid of altair plot containing all histograms
```

Comment:
You can't return two objects. Return a tuple of the two . Like tuple(pandas.DataFrame, altair)

Reply:
I changed it. Is it now correct?